deploy/post_deploy_testing/cypress/integration/03a_browse_views_basic.js

```diff
@@ -5,40 +5,42 @@ describe('Browse Views - Basic Tests', function () {
     context('Navigation and Redirection', function(){
 
         it('If start from home page, clicking on Browse nav menu item gets us to Browse page.', function(){
-
+            const dataNavBarItemSelectorStr = '#top-nav div.navbar-collapse .navbar-nav a.id-data-menu-item';
             cy.visit('/');
+            cy.get(dataNavBarItemSelectorStr).click().wait(500).then(()=>{
```

Comment:
Was this unfinished / forgot to push recent edits?
Looks like added new selector but not using it?

Was also gonna say maybe could put `dataNavBarItemSelectorStr` into `'./../support/variables';` so would be imported in same way as `navBrowseBtnSelector` was but this doesn't really matter unless need to use `dataNavBarItemSelectorStr` in some other files (which is possible / don't remember).

Reply:
`dataNavBarItemSelectorStr` selector is used only once, assigned as const is redundant. Removed.